music21/abcFormat/testFiles.py

```diff
@@ -838,7 +838,7 @@ def testAbc21Chords(self):
             s = translate.abcToStreamScore(ah)
             part = s.parts[0]
             self.assertFalse(part.getElementsByClass(chord.Chord),
-                             'Empty chord "%s" in Score' % abc_chord)
+                             f'Empty chord "{abc_chord}" in Score')
```

Comment:
great.  Most of these should be `{abc_chord!r}` (without the quotation marks) next time we refactor.